manufacturing/enclosed_build/display_panel/README.md

```diff
@@ -28,35 +49,54 @@ BEFORE purchasing any parts.**
-|**A9** | (4)       | McMaster-Carr | 92095A455                | 7.44 / 24    | [C][a9mcmc] | M2 screw 12mm, hex drive, **alternate to A8** |
-|**A10**| 4         | McMaster-Carr | 94180A351                | 15.47 / 100  | [C][a10mcmc]| Heat-set inserts for m4 screws |
-
-[a1rw]:    #upper-face-plate
+|**A4**| 1        | McMaster-Carr  | 5121K451            | 5.78 / 10    | [C][a4mcmc]   | 1/4 NPT x 1/4" ID barbed adapter |
+|**A5** | (1)       | UPerfect      | -                        | 49.00        | [A][a5ali]  | 7" capacitive touchscreen, **DUPLICATE in [electrical assembly][elec]** |
+|**A6** | 4         | McMaster-Carr | 93625A101                | 7.70 / 10    | [C][a6mcmc] | M2 locknut |
+|**A7** | 8         | McMaster-Carr | 93475A195                | 1.06 / 100   | [C][a7mcmc] | M2 washer, 5mm OD |
+|**A8** | (8)       | McMaster-Carr | 91116A240                | 2.40 / 100   | [C][a8mcmc] | M2 washer, 7mm OD, **alternate to A7** |
+|**A9** | 4         | McMaster-Carr | 92000A019                | 6.75 / 100   | [C][a9mcmc] | M2 screw 12mm, Phillip's drive |
+|**A10**| (4)       | McMaster-Carr | 92095A455                | 7.44 / 24    | [C][a10mcmc] | M2 screw 12mm, hex drive, **alternate to A9** |
+|**A11**| 4         | McMaster-Carr | 94180A351                | 15.47 / 100  | [C][a11mcmc]| Heat-set inserts for M4 screws |
+ 
+
+[a1rw]:    #custom-parts
 [a2mcmc]:  https://www.mcmaster.com/7768K22/
 [a3mcmc]:  https://www.mcmaster.com/50785K273/
-[a4ali]:   https://www.aliexpress.com/item/4000747984746.html
-[a5mcmc]:  https://www.mcmaster.com/93625A101/
-[a6mcmc]:  https://www.mcmaster.com/93475A195/
-[a7mcmc]:  https://www.mcmaster.com/91116A240/
-[a8mcmc]:  https://www.mcmaster.com/92000A019/
-[a9mcmc]:  https://www.mcmaster.com/92095A455/
-[a10mcmc]: https://www.mcmaster.com/94180A351/
+[a4mcmc]:  https://www.mcmaster.com/5121K451
+[a5ali]:   https://www.aliexpress.com/item/4000747984746.html
+[a6mcmc]:  https://www.mcmaster.com/93625A101/
+[a7mcmc]:  https://www.mcmaster.com/93475A195/
+[a8mcmc]:  https://www.mcmaster.com/91116A240/
+[a9mcmc]:  https://www.mcmaster.com/92000A019/
+[a10mcmc]: https://www.mcmaster.com/92095A455/
+[a11mcmc]: https://www.mcmaster.com/94180A351/
 
 ## Assembly Instructions
 
-The brass threaded check valve (**A2**) interfaces with a through-wall straight connector (**A3**) that will be on the
+### LCD Mounting
+Mounting the LCD on the acrylic cutout is fairly simple.
+1. Place the LCD in the cutout, face down.
+2. Insert the one washer (**A7**) into the screw (**A9**) and the screw-washer combination into one of the display corner holes.
+3. Insert the LCD-screw-washer combination into one of the display mounting holes.
+4. Insert a second washer (**A7**) and then the locknut (**A6**) from the other side of the acrylic. And fasten.
+5. Repeat for all four corners.
+### Oxygen Inlet System Installation
+To install the oxygen parts of the oxygen inlet system:
```

Comment:
Again, steps seem too much.
The really interesting things that are not here:
* which side of the panel is the lock washer on?
* do you need PTFE tape?